Plugins/NativeEngine/Source/ShaderCompilerTraversers.cpp

```diff
@@ -5,12 +5,15 @@
 #include <glslang/MachineIndependent/RemoveTree.h>
 
 #include <bgfx/bgfx.h>
+#include <bx/bx.h>
 
 #include <arcana/experimental/array.h>
 
 #include <gsl/gsl>
 
 #include <stdexcept>
+#include <iostream>
+#include <arcana/macros.h>
```

Comment:
What are these used for?

Reply:
iostream - unused, I'll remove this.

arcana/macros.h - I'm currently using their UNUSED(variable) macro. (L488)

Reply:
We haven't been consistently doing this, but using this seems like a good idea.